assets/careeropportunity/components/InfoOnOpportunity.js

```diff
@@ -67,6 +67,7 @@ const InfoBox = props => (
 
             <div className="row">
               <div className="col-md-12">
+                <p>Nettsted: <a href={props.website}> {props.website} </a></p>
```

Comment:
Can you add a screenshot of the new design to the PR?

Reply:
Done.